deploy/cluster-monitoring-config/cluster-monitoring-config.yaml

```diff
@@ -96,16 +96,7 @@ data:
         - effect: NoSchedule
           key: node-role.kubernetes.io/infra
           operator: Exists
-    # https://access.redhat.com/solutions/5685771
-    # https://bugzilla.redhat.com/show_bug.cgi?id=1906496
     thanosQuerier:
-      resources:
-        limits:
-          cpu: 500m
-          memory: 2048Mi
-        requests:
-          cpu: 5m
-          memory: 125Mi
```

Comment:
This config will apply to 4.6 clusters, which do not have the fix.

Reply:
4.6 clusters are no longer supported as of Aug 2021: https://docs.openshift.com/dedicated/osd_architecture/osd_policy/osd-life-cycle.html#rosa-life-cycle-dates_osd-life-cycle